yarn.lock

```diff
@@ -1572,11 +1572,6 @@ abab@^2.0.0:
   resolved "https://registry.yarnpkg.com/abab/-/abab-2.0.3.tgz#623e2075e02eb2d3f2475e49f99c91846467907a"
```

Comment:
Why doy ou changed `yarn.lock`?

Reply:
I don't know. Possible after calling `yarn`. I think you can save this change (tests pass correctly). If not, tell me and I'll fix it

Reply:
Just call `git checkout yarn.lock`